src/components/common/approval/TransactionDetailItem.tsx

```diff
@@ -12,6 +12,7 @@ import {
   type LinkItem,
   type TextItem,
   DetailItemType,
+  FundsRecipientItem,
```

Comment:
Seems like you have some import errors here.

Reply:
Yeah, I pinned the wrong version for the `VM-` packages. Corrected in ce3b826dc